broadpeakplugin/src/main/java/com/kaltura/playkit/plugins/broadpeak/BroadpeakPlugin.java

```diff
@@ -198,6 +199,13 @@ public void apply(PKMediaEntry mediaEntry, PKMediaEntryInterceptor.Listener list
             }
 
             PKMediaSource source = mediaEntry.getSources().get(0);
+            if (PKMediaFormat.udp.equals(source.getMediaFormat())) {
```

Comment:
do we also need to stop the session @giladna ? or because we return before we call `createStreamingSession` we don't even open a session?

Reply:
we do not open it at all so no need to close